### PR TITLE
Add links to k8s documentation + guide and cleanup

### PR DESCRIPTION
**Pull request by joelhans**
As stated in the title, this PR adds links to the new k8s documentation and guide that were both recently published. I also cleaned up the README a bit, and aligned some of the commands with those in the k8s install doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,61 @@
...
+To install the chart with the release name `netdata`:
 
 ```console
-$ helm install --name my-release ./netdata
+helm install netdata ./netdata-helmchart
 ```
 
-The command deploys nginx-ingress on the Kubernetes cluster in the default configuration.
-The [configuration](#configuration) section lists the parameters that can be configured during installation.
+The command deploys nginx-ingress on the Kubernetes cluster in the default configuration. The
```
Comment (knatsakis) [inline README.md:44]: Does it really deploy `nginx-ingress` ?

Comment (ilyam8) [reply README.md:44]: it is [ingress resource/object](https://github.com/netdata/helmchart/blob/master/templates/ingress.yaml), not the controller ofc.

i think `nginx-ingress => ingress`

Comment (prologic) [reply README.md:44]: @joelhans Please fix ☝️

Comment (joelhans) [reply README.md]: Done!

**Pull request merged**